azure-samples/src/main/java/com/microsoft/azure/management/samples/SSHShell.java

```diff
@@ -6,6 +6,9 @@
 
 package com.microsoft.azure.management.samples;
```

Comment:
The license header is in the first 5 lines of the file... This is an existing file and I only added the three methods I needed for the uploading/downloading/executing SSH commands.